apps/cowswap-frontend/src/modules/account/containers/Transaction/ActivityDetails.tsx

```diff
@@ -377,8 +378,11 @@ export function ActivityDetails(props: {
           {/* Order Currency Logo */}
           {inputToken && outputToken && (
             <ActivityVisual>
-              <TokenLogo token={inputToken} size={32} />
-              <TokenLogo token={isBridgeOrder ? swapAndBridgeOverview?.targetCurrency : outputToken} size={32} />
+              <CurrencyLogoPair
+                sellToken={inputToken}
+                buyToken={isBridgeOrder ? swapAndBridgeOverview?.targetCurrency || outputToken : outputToken}
```

Comment:
@shoom3301 added the `|| outputToken` fallback to handle the case where `swapAndBridgeOverview?.targetCurrency` is `undefined` (which fixes a TypeScript error I had, after merging in `develop`)